src/scripts/lib/server/utils/server-config/index.js

```diff
@@ -0,0 +1,26 @@
+const Configstore = require('configstore')
+const inq = require('inquirer')
+const prompts = require('./setup-prompts')
+
+module.exports = class ServerConfig {
+
+  constructor(defaults) {
+    this.db = new Configstore('wowser', defaults)
```

Comment:
Using the relative path approach to fetch `package.json` is probably alright:

```
const pkg = require('../../../../package.json')
```

Not entirely sure how many `../`s you'd need though :smile: